### PR TITLE
Fix/353/map rotation not working near cube

### PR DESCRIPTION
**Pull request by Med-H**
# fix map rotation not working near cube

Closes: #353 

## Possible improvements
The rotation speeds of codemap and viewcube differ. Viewcube makes a faster and a less smooth rotation since it's size is smaller compared to a codemap. Therefore we might want to set the rotation spped of cube dynamically through UI (see gifs).

## Screenshots and Gifs

**CodeMap rotation**
codemap_rotation

**ViewCube rotation**
viewcube_rotation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ## [unreleased] (Added 🚀 | Changed | Removed 🗑 | Fixed 🐞 | Chore 👨‍💻 👩‍💻)
```
Comment (ce-bo) [inline CHANGELOG.md:8]:
```suggestion
## [unreleased] (Added 🚀 | Changed | Removed 🗑 | Chore 👨‍💻 👩‍💻)
```

File: visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.ts [hunk 2]
```diff
@@ -248,9 +253,24 @@ export class CodeMapMouseEventService
 		return labelForBuilding
 	}
 
+	private EnableOrbitalsRotation(isRotation: boolean) {
+		this.threeViewerService.enableRotation(isRotation)
+		this.viewCubeMouseEventsService.enableRotation(isRotation)
+	}
+
+	onDocumentMouseEnter() {
+		this.EnableOrbitalsRotation(true)
```
Comment (ce-bo) [inline visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.ts:262]: I am not sure about the code style. I assume that functions/methods should be written camelCase.

**Review by ce-bo — APPROVED** [review]
Mainly LGTM.

File: visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.ts (hunk 2 above)
Comment (Med-H) [reply visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.ts:262]: oops :)

**Review by RomanenkoVladimir — APPROVED** [review]

**Pull request merged**